### PR TITLE
Youtube layout

### PR DESCRIPTION
**Pull request by AndreasGalster**
### Description
This is the first YouTube HTML/CSS project

HTML Validation and CSS linting has been implemented

### CODE REVIEW

**Comment by arslanbisharat** [comment]
Friendly reminder: You should make sure that your repo looks professional before review.
Please take a step back and read a lesson here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.
This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,606 @@
...
+              <p>It was really good to see this video man.</p>
+              <div>
+                <button>
+                  <img alt='Like comment' src='./assets/icon-thumbs-up.svg'>
+                  4
+                </button>
+                <button>
+                  <img alt='Dislike comment' src='./assets/icon-thumbs-down.svg'>
+                  1
+                </button>
```
Comment (arslanbisharat) [inline index.html]: add header once in  code

File: index.html [hunk 2]
```diff
@@ -0,0 +1,606 @@
...
+              </button>
+              <button id='playlist-button'>
+                <img class='icon' src='./assets/icon-playlist.svg' alt='Save to playlist'>
+                SAVE
+              </button>
+
+            </div>
+          </header>
+
+          <section id='description'>
```
Comment (arslanbisharat) [inline index.html]: add main after header

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes
I can see you worked hard on this project but I am sorry to inform you that I cannot Approve this project because it is not completed yet.😟
_It's okay. We are here to learn. Don't get discouraged. Work more hard._
**Please have a look at the suggestions to get more understanding. 👀**
- [ ] Please make your readme more descriptive. 

- [ ] Add a live link to your project. You can use this link [Click here](https://raw.githack.com/)

- [ ] You have many CSS files Please try to add all of your styling code in a single file because this makes the project better.

- [ ] This section doesn't look correct Please work on it
chrome-capture (14)

- [ ] Please remove comments that are not necessary 

- [ ]  Use systematic tag (main, header, and footer) when required and only once. 

**Please make the changes and request re-review for the given project**
Stay safe😇

### Happy Coding
-------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Comment by AndreasGalster** [comment]
Hi @arslanbisharat thanks for the feedback! We'll get to it shortly.

I have a few thoughts/questions on the following points:

> * [ ]  You have many CSS files Please try to add all of your styling code in a single file because this makes the project better.

I'm not sure I agree with you on this one. Technically that is Ok to do since our project is relatively small but especially on bigger projects (e. g. sass) you'd typically import many files to keep things clean. Since we're not using components (React, webcomponents, etc), I believe this makes it easier to find the correct styles to edit.

If you really think the change is necessary I'm OK to do it although especially in peer programming keeping our CSS structured to make it easier to find our way around the CSS is pretty helpful. It's quite overwhelming to look at a very long CSS file that partly wasn't written by you.

I agree that this would probably cause issues with .css files on a simple project (multiple parallell downloads), but with a typical build process (.scss, webpack) it would end up in one file before going to production.

https://webdesign.tutsplus.com/articles/css-documentation-best-practices--cms-30139

Number 2 and number 4 explain the issue with long stylesheets.

> * [ ]  Please remove validation errors. If you don't know anything about it please click [here](https://validator.w3.org/)

Can you please explain which errors you'd like fixed? We used the html-validator-cli as described in the GitHub documentation to fix our errors.

image

validator.w3.org also doesn't throw any errors:

image


> **Please make the changes and request re-review for the given project**
> Stay safe😇

Stay safe as well 😃

**Comment by arslanbisharat** [comment]
@AndreasGalster  I am sorry for my mistake. I talked to my senior and they informed me it's okay to use too many CSS files but there is no organization in your CSS files. Please add them into any folder(assets) 
About Validation Please ignore it. I removed it from my previous comment. Your validation is fine that was on my side.

**Comment by AndreasGalster** [comment]
Hi @arslanbisharat we've just fixed all the issues as you suggested. Please let us know if this is good to go now 🙏

**Review by arslanbisharat — APPROVED** [review]
### Congratulations APPROVED :clap:
Hello I can see you worked really hard on this project. and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
_The clone shows a good understanding and usage of HTML CSS 💯_

- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

- [ ] **Optional** Please add description in about section of your repo. 
chrome-capture (47)

- [ ] **Optional** Please add more social links like Linkedin and Twitter in your readme. 
chrome-capture (46)

- [ ] **Optional** Please add css file into its folder
chrome-capture (48)




[**OPTIONAL**] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**